vochain/scrutinizer/vote.go

```diff
@@ -174,6 +182,17 @@ func (s *Scrutinizer) addLiveVote(envelope *models.Vote, results *Results) error
 	return nil
 }
 
+// addVoteIndex adds the nullifier reference to the kv for fetching vote Txs from BlockStore.
+// This method is triggered by Commit callback for each vote added to the blockchain.
+func (s *Scrutinizer) addVoteIndex(nullifier, pid []byte, blockHeight int64, txIndex int32) error {
+	_, err := s.VochainState.Process(pid, false)
```

Comment:
There is no need for this call. We should in general try to minimize the number of database transactions.